cockroachdb/values.yaml

```diff
@@ -134,6 +134,16 @@ conf:
   # CockroachDB's port to listen to HTTP requests.
   http-port: 8080
 
+  # CockroachDB's storage configuration https://www.cockroachlabs.com/docs/v20.2/cockroach-start.html#storage
+  store:
+    enabled: false
```

Comment:
It's a little strange that `enabled` can be `false`, but that the `store.path` field is still read for the `mountPath`.  Maybe `store.path` should be moved to the top level of this config?